snippets/visualbasic/VS_Snippets_CLR/Generic.LinkedList/vb/source.vb

```diff
@@ -157,48 +157,48 @@ End Class
 'This code example produces the following output:
 '
 'The linked list values:
-'the fox jumped over the dog 
+'the fox jumps over the dog 
 'Test 1: Add 'today' to beginning of the list:
-'today the fox jumped over the dog 
+'today the fox jumps over the dog 
 'Test 2: Move first node to be last node:
-'the fox jumped over the dog today 
+'the fox jumps over the dog today 
 'Test 3: Change the last node to 'yesterday':
-'the fox jumped over the dog yesterday 
+'the fox jumps over the dog yesterday 
```

Comment:
This is the only change that we might want to update. "The fox jumps over the dog yesterday" just sounds like it should be past tense.  Maybe the simplest change would be to replace "yesterday" with "later" so the grammar is correct.